src/interfaces.ts

```diff
@@ -8,34 +8,41 @@ export type LogFile = MergedLogFile | ProcessedLogFile;
 
 export type RepeatedCounts = Record<string, number>;
 
-export const enum LogType {
-  TRACE = 'trace',
+export enum LogType {
+  ALL = 'all',
   BROWSER = 'browser',
   RENDERER = 'renderer',
-  CALL = 'call',
-  WEBAPP = 'webapp',
   PRELOAD = 'preload',
+  WEBAPP = 'webapp',
+  STATE = 'state',
+  CALL = 'call',
   NETLOG = 'netlog',
+  TRACE = 'trace',
   INSTALLER = 'installer',
-  ALL = 'all',
   MOBILE = 'mobile',
   CHROMIUM = 'chromium',
-  UNKNOWN = ''
+  UNKNOWN = 'unknown',
 }
 
-export const ALL_LOG_TYPES = [
+export type KnownLogType = Exclude<LogType, LogType.UNKNOWN | LogType.ALL>;
+export type SelectableLogType = Exclude<LogType, LogType.UNKNOWN>;
+
+export type ProcessedLogFiles = Record<KnownLogType, Array<ProcessedLogFile>>;
```

Comment:
For some of the types, it would make them `Array<ProcessedLogFile>` which is incorrect as they are suppose to be `Array<UnzippedFile>`

Reply:
See [d538582](https://github.com/tinyspeck/sleuth/pull/84/commits/d538582cd257b9728a9693468fff2ad8d04add69)